component/scripts/approve.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+LAST_UPGRADE_JOB_TIMESTAMP=$(kubectl -n "${UPGRADE_CONTROLLER_NAMESPACE}" get upgradejobs.managedupgrade.appuio.io -ojson | jq -r '.items | sort_by(.metadata.creationTimestamp) | reverse | .[0] | .metadata.creationTimestamp')
```

Comment:
all UpgradeJobHooks have environment variables containing all relevant UpgradeJob information, including the creation timestamp, so this is completely unnecessary.

```suggestion
```